lib/timers.js

```diff
@@ -160,16 +160,51 @@ function listOnTimeout() {
   debug('timeout callback %d', msecs);
 
   var now = TimerWrap.now();
-  debug('now: %s', now);
+  debug('now: %d', now);
 
-  var diff, timer;
+  var diff, timer, timeRemaining;
```

Comment:
But there are two places that use the same variable.  I guess that might not be accurate now that you're suggesting the first block isn't necessary.